fn main() {
    let x = 25;
    println!("The value of x is: {}", x);
    x = 75; //This is throwing error. See README for details
    println!("The value of x is: {}", x);
}
